linux/datadog_wrapper

```diff
@@ -67,30 +84,78 @@ setUpNodeEnv() {
 setUpDotnetEnv() {
     echo "Setting up Datadog tracing for .NET"
 
-    setUpCommonEnv || return
+    getBinaries || return
```

Comment:
These `getBinaries` calls should be moved to the common method because we will need to call it for all languages.